test/boc.jl

```diff
@@ -1,6 +1,5 @@
 @testset "BOCcos" begin
-
-    @testset "BOCcos($system, 0, $n)" for system in [GPSL1()], n in [1]#[GPSL1(), GPSL5(), GalileoE1B()], n in [1, 5, 10]
+    @testset "BOCCos($(get_system_string(system)), 0, $n)" for system in [GPSL1()], n in [1]#[GPSL1(), GPSL5(), GalileoE1B()], n in [1, 5, 10]
```

Comment:
If `get_system_string` is not exported anymore, this should be `GNSSSignals.get_system_string`